microreactors/mrad/heat_pipe_failure/HPMR_thermo_tr.i

```diff
@@ -0,0 +1,431 @@
+    multi_app = sockeye
+    source_variable = hp_temp_aux
+    variable = hp_temp_aux
+    execute_on = 'initial timestep_begin'
+    fixed_meshes = true
+  []
+  [to_sockeye_flux]
+    type = MultiAppNearestNodeTransfer
+    direction = to_multiapp
+    multi_app = sockeye
+    source_variable = flux_uo_corr
+    variable = master_flux
+    execute_on = 'initial timestep_begin'
+    fixed_meshes = true
+  []
+  [send]
+    type = MultiAppVectorPostprocessorTransfer
+    vector_postprocessor = reader
+    postprocessor = htc_receive
+    vector_name = htc_hp
+    direction = to_multiapp
+    multi_app = sockeye
+    execute_on = 'initial timestep_begin'
+  []
+[]
+
+[UserObjects]
+  [flux_uo]
+    type = NearestPointLayeredSideFluxAverage
+    direction = z
+    num_layers = 100
+    points_file = 'hp_centers.txt'
+    variable = temp
+    diffusivity = thermal_conductivity
+    execute_on = linear
+    boundary = 'heat_pipe_ht_surf'
+  []
+[]
+
+[Executioner]
+  type = Transient
+
+  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart '
+  petsc_options_value = 'hypre boomeramg 100'
+
+  nl_rel_tol = 1e-6
+  nl_abs_tol = 1e-8
+
+  start_time = 0
+  end_time = 2000
+  dt = 1
+[]
+
+[VectorPostprocessors]
+  [reader]
+    type = CSVReader
+    csv_file = 'hp_params.data'
+    header = true
+    allow_duplicate_execution_on_initial = true
+  []
+[]
+
+[Postprocessors]
+  [hp_heat_integral]
+    type = SideFluxIntegral
+    variable = temp
+    boundary = 'heat_pipe_ht_surf'
+    diffusivity = thermal_conductivity
+    execute_on = 'initial timestep_end'
+  []
+  [fuel_temp_avg]
+    type = ElementAverageValue
+    variable = temp
+    block = 'fuel_quad fuel_tri'
+  []
+  [fuel_temp_max]
+    type = ElementExtremeValue
+    variable = temp
+    block = 'fuel_quad fuel_tri'
+  []
+  [fuel_temp_min]
+    type = ElementExtremeValue
+    variable = temp
+    block = 'fuel_quad fuel_tri'
+    value_type = min
+  []
+  [mod_temp_avg]
+    type = ElementAverageValue
+    variable = temp
+    block = 'moderator_quad moderator_tri'
+  []
+  [mod_temp_max]
+    type = ElementExtremeValue
+    variable = temp
+    block = 'moderator_quad moderator_tri'
+  []
+  [mod_temp_min]
+    type = ElementExtremeValue
+    variable = temp
+    block = 'moderator_quad moderator_tri'
+    value_type = min
+  []
+  [monolith_temp_avg]
+    type = ElementAverageValue
+    variable = temp
+    block = monolith
+  []
+  [monolith_temp_max]
+    type = ElementExtremeValue
+    variable = temp
+    block = monolith
+  []
+  [monolith_temp_min]
+    type = ElementExtremeValue
+    variable = temp
+    block = monolith
+    value_type = min
+  []
+  [heatpipe_surface_temp_avg]
+    type = SideAverageValue
+    variable = temp
+    boundary = heat_pipe_ht_surf
+  []
+  [power_density]
+    type = ElementIntegralVariablePostprocessor
+    block = 'fuel_quad fuel_tri'
+    variable = power_density
+    execute_on = 'initial timestep_end transfer'
+  []
```

Comment:
This is power since it's an integral of power density right?

I m thinking of renaming it

Reply:
You are right. This is a pp from a legacy input (I was too lazy to rename it...).